### ISSUE TITLE
All files open as empty buffers via gvfs on Linux

### ISSUE THREAD

**Issue by khaeru** [atom/atom#11784]
### Description

I am trying to use Atom to work on files on a machine I am connected to via SFTP/SSH.

### Steps to Reproduce

1. Using Ubuntu, connect to a remote machine (e.g. via SFTP) using the file manager.
2. Open Atom.
3. "Open Folder…" and select a directory on the remote machine in the dialog. (Here the file tree will correctly show the files and directories on the remote machine, and the window title will include something like `/run/user/1000/gvfs/sftp:host=svante.mit.edu/home/pnk/igsm`.)
4. Open a file from the file tree.

**Expected behavior:** Contents of the file are shown.

**Actual behavior:** An empty buffer is shown.

### Versions
```
$ uname -a && lsb_release -rd && apt-cache policy atom && atom --version && apm --version
Linux khaeru-laptop 4.4.0-22-generic #39-Ubuntu SMP Thu May 5 16:53:32 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
Description:    Ubuntu 16.04 LTS
Release:        16.04
atom:
  Installed: 1.7.3-1~webupd8~0
  Candidate: 1.7.3-1~webupd8~0
  Version table:
 *** 1.7.3-1~webupd8~0 500
        500 http://ppa.launchpad.net/webupd8team/atom/ubuntu xenial/main amd64 Packages
        100 /var/lib/dpkg/status
Atom    : 1.7.3
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
apm  1.9.2
npm  2.13.3
node 0.10.40
python 2.7.11+
git 2.7.4
``

**Comment by 50Wliu** [comment]
I believe this issue has been fixed in the Atom 1.8.0 betas, so I'm going to close this.  If that's not the case though, please comment here!

**Comment by khaeru** [comment]
Confirmed fixed in 1.8.0.
